plugins/pencilblue/templates/angular/admin/content/pages/manage_pages.html

```diff
@@ -34,41 +42,116 @@
         }, {
             name: loc.pages.PUBLISH_DATE,
             field: 'publish_date',
-            sortAsc: true,
-            sortDesc: false
+            sortAsc: false,
+            sortDesc: true
         }];
 
+        $scope.getPages = function(cb) {
+          if(!cb) {
+            cb = function() {};
+          }
+
+          $scope.pages = null;
+
+          pageFactory.getPages({
+            $limit: $scope.paginationLimit,
+            $offset: $scope.paginationIndex,
+    				$order: $scope.orderString
+          }, function(error, pages, total) {
+            if(error) {
+              $scope.errorMessage = error.message;
+              return;
+            }
+
+            $scope.pages = $scope.setPageStatuses(pages);
+            $scope.paginationTotal = total;
+            cb(pages, total);
+          });
+        };
+
+    		$scope.setPageStatuses = function(pages) {
+    			var now = new Date();
+    			for(var i = 0; i < pages.length; i++) {
+    				if(pages[i].draft) {
+    					pages[i].status = loc.articles.DRAFT
+    				}
+    				else if(pages[i].publish_date > now) {
+    					pages[i].status = loc.articles.UNPUBLISHED;
+    				}
+    				else {
+    					pages[i].status = loc.articles.PUBLISHED;
+    				}
+    			}
+
+    			return pages;
+    		}
```

Comment:
The code to handle articles and pages is really similar.  How hard would it be to condense this down so we don't have duplicate code?